addons/mail/static/src/backend_components/activity_list_view_item/activity_list_view_item.xml

```diff
@@ -10,7 +10,7 @@
                 <t t-if="!activityListViewItem.activity.summary and activityListViewItem.activity.type">
                     <b class="text-900 me-2 text-truncate" t-esc="activityListViewItem.activity.type.displayName"/>
                 </t>
-                <button t-if="activityListViewItem.activity.chaining_type === 'suggest' and activityListViewItem.activity.canWrite" class="btn btn-sm btn-link" t-on-click="activityListViewItem.onClickEditActivityButton">
+                <button t-if="activityListViewItem.hasEditButton and activityListViewItem.activity.chaining_type === 'suggest' and activityListViewItem.activity.canWrite" class="btn btn-sm btn-link" t-on-click="activityListViewItem.onClickEditActivityButton">
```

Comment:
If a field is introduced it should contain the other part of the condition `activityListViewItem.activity.chaining_type === 'suggest' and activityListViewItem.activity.canWrite`